docs/community.aws.ecs_taskdefinition_module.rst

```diff
@@ -26,9 +26,9 @@ Requirements
 The below requirements are needed on the host that executes this module.
 
 - boto3
-- boto3 >= 1.15.0
+- boto3 >= 1.16.0
 - botocore
```

Comment:
Why do we still have here `botocore` since we require `botocore >= 1.19.0` in the next?

Reply:
I _guess_ because https://github.com/ansible-collections/community.aws/pull/809 was not backported

Reply:
I'm not sure it is also required for a 2.x release?  
cc @tremble @jillr

Reply:
hm no, I've got no idea atm

Reply:
https://gist.github.com/markuman/d1162ef37a88be0562ba1cada3893e2e

Reply:
https://github.com/ansible-collections/community.aws/pull/863